src/components/ui/coaching-sessions/coaching-notes/toolbar.tsx

```diff
@@ -25,6 +25,18 @@ export const Toolbar = () => {
   if (!editor) {
     return null;
   }
+  // Add keyboard shortcut handler
+  useEffect(() => {
+    const handleKeyDown = (e: KeyboardEvent) => {
+      if ((e.metaKey || e.ctrlKey) && e.key === "k") {
```

Comment:
This isn't working today because of our command menu at the top, it listens for CMD + k as well which is in conflict. We do want to hide/remove that search for the beta anyway, so if you do that, this should work.

I changed the key combination to listen for 'l' and then CTRL + l worked for me.